prow_config.yaml

```diff
@@ -26,19 +26,35 @@ workflows:
       - deployment/*
     params:
       platform: gke
+      gkeApiVersion: v1
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: kfctl_test
+    name: kfctl-beta
+    job_types:
+      - presubmit
```

Comment:
Per you comment; I thought we didn't want to test v1beta as a presubmit? 

Would it make sense to only test v1beta1 on presubmit if the GKE configs change?

Reply:
Changed - scoped this to when gke scripts or config files are changed.